agent/app/agent_capability.go

```diff
@@ -63,7 +66,12 @@ func (agent *ecsAgent) capabilities() []*ecs.Attribute {
 	// Determine API versions to report as supported. Supported versions are also used for capability-enablement, except
 	// logging drivers.
 	for _, version := range agent.dockerClient.SupportedVersions() {
-		capabilities = appendNameOnlyAttribute(capabilities, capabilityPrefix+"docker-remote-api."+string(version))
+		dockerVersion, _ := strconv.ParseFloat(string(version), 64)
+		if dockerVersion >= dockerVersionUsingNewCapabilityPrefix {
```

Comment:
Why are we doing this?

Reply:
I believe the new agent capabilities have the "ecs.capability" prefix. Shouldn't we maintain the same for new remote api versions too?

Reply:
I think we should leave all the api version capabilities consistent with each other.